src/scripts/main.js

```diff
@@ -1,3 +1,312 @@
 'use strict';
 
-// write your code here
+const start = document.querySelector('.start');
+const restart = document.querySelector('.restart');
+const tableRows = document.querySelectorAll('tr');
+const score = document.querySelector('.game-score');
+const messageContainer = document.querySelector('.message-container');
+const startMessage = document.querySelector('.message-start');
+const loseMessage = document.querySelector('.message-lose');
+const winMessage = document.querySelector('.message-win');
+
+const isMobile = /iPhone|iPad|iPod|Android/i.test(navigator.userAgent);
```

Comment:
It's very nice that you made adaptation for mobile devices also.

P.s.: point to to improve, it's remove scrollbar for mobile because now a little bit uncomfortable to play due to scrolling.